src/envo/envo-edit.owl

```diff
@@ -37280,6 +37356,91 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000140> "f
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000144> (natural gas processing plant)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Natural-gas_processing") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000144> "A refinery which is capable of processing natural gas. Natural-gas processing encompasses a range of industrial processes designed to purify raw natural gas by removing impurities, contaminants and higher molecular mass hydrocarbons to produce what is known as pipeline quality dry natural gas")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000144> <https://orcid.org/0000-0002-6601-2165>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000144> "natural gas processing plant"@en)
+EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_02000144> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_02000141> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002215> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_02000147> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002233> <http://purl.obolibrary.org/obo/ENVO_01000552>)))))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000144> <http://purl.obolibrary.org/obo/ENVO_02000141>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000146> (chemical engineering process)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Chemical_engineering") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://www.britannica.com/technology/chemical-engineering") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000146> "A planned process which has the intention of producing, consuming, or transforming materials via the execution of chemical reactions.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000146> "cjm"^^xsd:string)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_02000146> "industrial chemical processing")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000146> "chemical engineering process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000146> <http://purl.obolibrary.org/obo/OBI_0000011>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000147> (refinement process)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Refining") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000147> "A planned material transformation process which takes as input a material and removes impurities.")
```

Comment:
>A planned material transformation process which takes as input a material and removes impurities

Do we define impurities anywhere? If not, something showing an active designation would be useful:

>A planned material transformation process during which material designated as impurities are removed from an material taken as input.